docs/guides/connecting_to_your_data/cloud/azure/spark.md

```diff
@@ -0,0 +1,145 @@
+---
+title: How to connect to data on Azure Blob Storage using Spark
+---
+
+import NextSteps from '../../components/next_steps.md'
+import Congratulations from '../../components/congratulations.md'
+import Prerequisites from '../../components/prerequisites.jsx'
+import WhereToRunCode from '../../components/where_to_run_code.md'
+import SparkDataContextNote from '../../components/spark_data_context_note.md'
+import SparkAdditionalNotes from '../../components/spark_additional_notes.md'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This guide will help you connect to your data stored on Microsoft Azure Blob Storage (ABS) using Spark.
+This will allow you to validate and explore your data.
+
+<Prerequisites>
+
+- Have access to data on an ABS container
```

Comment:
one very small nitpick is to have this be `Have access to data on an ABS bucket
`, which would match the Pandas doc

Reply:
@Shinnnyshinshin I have to fix the `Pandas` document then -- because the "bucket" for Azure is "container".  Thank you for catching this.

Reply:
@Shinnnyshinshin Fixed as per the comment.  Thanks!